src/asciidoc/docs/develop/addService.adoc

```diff
@@ -193,54 +189,126 @@ app.listen(process.env.PORT || 3333, null, async err => {
+You must update the `koji.json` file to let Koji know about the new service.
+`koji.json` is located in the root directory.
 
-. Open the `.koji/project/develop.json` file and update it to look like the following code.
+. Open the `koji.json` file, and replace the existing content with the following code.
 +
 [source,json]
 ----
-include::../_includes/develop.json[tags=**,indent=0]
+{
+  "deploy": {
+    "frontend": {
+      "output": "frontend/build",
+      "type": "static",
+      "commands": [
+        "cd frontend",
+        "npm install",
+        "export NODE_ENV=production && npm run build"
+      ]
+    },
+    "backend": {
+      "output": "backend",
+      "type": "dynamic",
+      "commands": [
+        "cd backend",
+        "npm install",
+        "export NODE_ENV=production && npm run compile"
+      ]
+    }
+  },
+  "develop": {
+    "frontend": {
+      "path": "frontend",
+      "port": 8080,
+      "startCommand": "npm start"
+    },
+    "backend": {
+      "path": "backend",
+      "port": 3333,
+      "startCommand": "npm run start-dev"
+    }
+  },
+  "entitlements": {
+    "InstantRemixing": true,
+    "InstantRemixingNativeNavigation": true,
+    "FeedEvents": true
+  },
+  "remixData": {
+    "backgroundColor": "#282c34"
+  },
+  "@@initialTransform": {
+    "remixData": {}
+  }
+}
 ----
 +
-This file lets the Koji editor know which services to run, so you must add an entry for the `backend` service.
+This file lets the Koji editor know which services to run, so you must add entries for the `backend` service.
 . On the left side of the editor, go to *Advanced > Remote environment* and click *Force restart project*.
 +
 If everything went well, you should be able to reconnect to the editor and see that a `backend` terminal has been added at the bottom.
 +
 If you see `[koji] backend started`, your backend service is now running in your local editor environment.
-. To confirm that the `serviceMap` is now populated, open your preview tab and refresh it. You should now see a console log that looks like this example.
+
+== Testing the backend service
+
+To test that everything is working correctly, you can modify your frontend code to get a value from the backend service and display it automatically.
+
+. Go to the `frontend` folder, press CTL-C to stop the frontend server, and install the `styled-components` package, which the new code will use.
 +
-[source,console]
+[source,bash]
 ----
-serviceMap: {
-  frontend: "https://8080-984c0a6f-3614-4dee-89f0-7d7cc03df6e9.koji-staging.com",
-  backend: "https://3333-984c0a6f-3614-4dee-89f0-7d7cc03df6e9.koji-staging.com"}
+npm install styled-components
 ----
-. Open the `.koji/project/deploy.json` file and update it to look like the following code.
+. Restart the frontend server.
 +
-[source,json]
+[source,bash]
 ----
-include::../_includes/deploy.json[tags=**,indent=0]
+npm run start-dev
```

Comment:
Isn’t this the command to start the backend server? If we have them use the frontend terminal to install the styled-components package, I believe we can change this line to `npm start`.